provisioning/device/src/constants.ts

```diff
@@ -28,5 +28,5 @@ export class ProvisioningDeviceConstants {
   /**
    * default timeout to use when communicating with the service
    */
-  static defaultTimeoutInterval: number = 4000;
+  static defaultTimeoutInterval: number = 30000;
```

Comment:
>30000 [](start = 42, length = 5)

just double checking if this is vetted with other SDKs and the service folks - this is an important value.